api/src/opentrons/protocol_engine/execution/heater_shaker_movement_flagger.py

```diff
@@ -0,0 +1,55 @@
+"""Helpers for flagging unsafe movements around a heater-shaker Module."""
+
+from typing import List
+
+from opentrons.protocols.geometry.deck_conflict import (
+    get_east_west_locations,
+    get_north_south_locations,
+)
+
+from ..errors import RestrictedPipetteMovementError
+from ..types import HeaterShakerMovementData
+
+
+async def raise_if_movement_restricted_by_heater_shaker(
+    heater_shaker_data: List[HeaterShakerMovementData],
+    destination_slot: int,
+    is_multi_channel: bool,
+    is_tiprack: bool,
```

Comment:
Suggested rename:

```suggestion
    destination_is_tip_rack: bool,
```

("Destination" for clarity. "Tip rack" being two words is a pedantic pet peeve of mine.)